components/camel-flink/src/main/java/org/apache/camel/component/flink/FlinkConstants.java

```diff
@@ -14,15 +14,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package org.apache.camel.component.flink;
 
-public final class FlinkConstants {
+public class FlinkConstants {
 
-    public static final String FLINK_DATASET_HEADER = "CamelFlinkDataSet";
+    public static final String FLINK_DATASET_HEADER = "CAMEL_FLINK_DATASET";
```

Comment:
The standard for keys in Camel is as in that red line, eg camelComponentNameKeyName